pkg/api/validation/validation.go

```diff
@@ -1921,6 +1921,10 @@ func ValidateSecurityContext(sc *api.SecurityContext) errs.ValidationErrorList {
 			allErrs = append(allErrs, errs.NewFieldInvalid("runAsUser", *sc.RunAsUser, "runAsUser cannot be negative"))
 		}
 	}
+
+	if sc.SELinuxOptions != nil && !capabilities.Get().EnableSELinuxIntegration {
+		allErrs = append(allErrs, errs.NewFieldForbidden("selinuxOptions", "Cannot specify selinuxOptions if SELinux integration is not enabled"))
```

Comment:
@pweil- The presumption here is that if the user specifies an SELinux context then they want the container to run under that context. If SELinux is disabled then the pod will run of course but it will be less restricted than the user desired it to be.